ui/site/src/forum.ts

```diff
@@ -33,7 +33,9 @@ lichess.load.then(() => {
     $(el).replaceWith($('.forum-post__message', el));
   });
   $('.forum-post__message').each(function (this: HTMLElement) {
-    if (this.innerText.match(/(^|\n)>/)) {
+    const isVisible = window.getComputedStyle(this).display !== 'none';
```

Comment:
not a fan of `getComputedStyle`, it's hacky and slow - and runs on each blog post. Can we test a class instead?